spec/controllers/proposals_controller_spec.rb

```diff
@@ -279,7 +278,8 @@
       flash.clear
       post :approve, id: proposal.id
 
-      expect(response.status).to eq(403)
+      expect(response).to redirect_to(proposal_path(proposal))
+      expect(flash[:error]).not_to be_nil
```

Comment:
sure thing. I was trying to not tie the test too closely to the exact error wording, but I guess the controller action already depends on an exact string match (which I find unfortunate but couldn't see an easy way around it).